mora/service/itsystem.py

```diff
@@ -223,19 +228,33 @@ def get_relation_for(value, start, end):
         }
 
     def create(self, id, req):
+        systemobj = common.checked_get(req, keys.ITSYSTEM, {})
+        systemid = common.get_uuid(systemobj)
+
+        if not systemid:
+            raise ValueError('missing system uuid!')
+
+        if not util.is_uuid(systemid):
+            raise ValueError('invalid system uuid!')
```

Comment:
I don't think we'll ever reach this branch... All the checks happen in the `get_uuid`/`checked_get` calls.

Reply:
Hmmm, nice catch, I'll see if this code is dead, and drop it if so!

Reply:
And the same goes for the check right above it, I think.